### ISSUE TITLE
Nim int-to-text precedence

### ISSUE THREAD

**Issue by MeWhenI** [jared-hughes/polygolf#45]
Compiling
```
$b:0..1 <- 1;
$a <- (int_to_text (1 + $b));
```
to Nim produces
```Nim
var
 b=1
 a= $1+b
```
This is a syntax error. It should produce
```Nim
var
 b=1
 a= $(1+b)
```

**Comment by MeWhenI** [comment]
Potentially fixed by #41

**Comment by MichalMarsalek** [comment]
> Potentially fixed by #41

It indeed is.
